testing-modules/junit-5/src/test/java/com/baeldung/groupedvsmultipleassertions/AssertAllUnitTest.java

```diff
@@ -0,0 +1,19 @@
+package com.baeldung.groupedvsmultipleassertions;
+
+import groupedvsmultipleassertions.User;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class AssertAllUnitTest {
+    @Test
+    void givenAssertAll_whenSingleAssertStatementFails_thenRestWillStillBeExecuted() {
+        User user = new User("baeldung", "admin@baeldung.com", true);
+        assertAll(
+                "Exploring assertAll behavior",
+                () -> assertEquals("baeldung", user.getUsername(), "Username should be baeldung"),
+                () -> assertEquals("admin@baeldung.com", user.getEmail(), "Email should be admin@baeldung.com"),
+                () -> assertTrue(user.getActivated(), "User should be activated")
+        );
+    }
```

Comment:
Can we see a failing test> We can `@Disabled` it so it won't break the test pack.